applications/PGSAUT-abstract.md

```diff
@@ -0,0 +1,53 @@
+# Abstract for The PGSA@UT Philosophy Conference
+
+### Abstract for submission to The [Philosophy Graduate Student Association at The University of Tennessee Philosophy Conference](http://utkphilgradconference.weebly.com/) March 13-14 2021
+
+## Hacking's Human Kinds and the Ethics of Participatory Science 
+
+Drawing on Hacking’s ([1996](doi.org/\10.1093\acprof:oso\9780198524021.001.0001) influential concept of the ‘looping effects of human kinds’, this paper will explore the ethical implications of the dynamics between researchers and research subjects in scientific practice. 
+We argue that scientists engaged in work about human kinds have a moral duty to include research subjects in their work. 
+We will then ground the theoretical discussion with a case study of participatory science in action: a project to build a tool to investigate autism and sensory processing through the participatory co-creation of a citizen science platform. 
+This project is a collaboration between the research charity Autistica and The Alan Turing Institute, the UK’s national institute for AI and data science. 
+More detail, current work, outputs, resources and engagement pathways are all available on the [project’s repository](https://github.com/alan-turing-institute/AutisticaCitizenScience).
+
+The presentation of this paper will itself follow participatory practices: it will be co-presented by an autistic participant, point attendees to open and transparent research protocols and data, and everyone will be invited to join as collaborators.
+
+We begin by exploring how existing scientific practices base themselves on a division between researchers and research subjects. 
+To do so, we explore Hacking’s account of human kinds and provide the example of autism. 
+We briefly review recent trends in the study of autism and argue that the gap traditionally conceived between “knowers” and those who are “known” raises ethical concerns.
+
+The ethical concerns we discuss are similar to those of feminist scholars at large and standpoint theorists in particular. 
+We will consider what Miranda Fricker ([2007](doi.org/10.1093/acprof:oso/9780198237907.001.0001)) calls “epistemic injustice”. 
+By Fricker’s account, prejudice about certain stereotypes render their testimony noncredible. 
+When these stereotypes are incorrect, an epistemic injustice is produced. 
+This is because social injustices are perpetuated through incorrectly assigned stereotypes. Despite the research subject being an expert of the own experience, research findings are sanctioned by scientific institutions and afforded authority by society. 
```

Comment:
```suggestion
This is because social injustices are perpetuated through incorrectly assigned stereotypes. Despite the research subject being an expert of the own experience, research findings are recognised by scientific institutions and afforded authority by society. 
```

Reply:
I suggest removing the word 'sanctioned' because it has two contradictory meanings.

Reply:
Yeah, sanctioned is such a confusing one. For the longest time I thought there was only one meaning of 'sanctioned' which for some reason I could never remember. Realising it actually has 2 meanings was totally shocking, like discovering someone you thought is one person is actually two identical twins.